src/FIRO_CFP_Plugin.java

```diff
@@ -1,23 +1,45 @@
 import com.rma.factories.NewObjectFactory;
 import hec2.map.GraphicElement;
 import hec2.model.DataLocation;
+import hec2.model.ProgramOrderItem;
 import hec2.plugin.CreatablePlugin;
 import hec2.plugin.action.EditAction;
 import hec2.plugin.action.OutputElement;
 import hec2.plugin.lang.ModelLinkingException;
 import hec2.plugin.lang.OutputException;
 import hec2.plugin.model.ModelAlternative;
+import hec2.plugin.model.ComputeOptions;
 import hec2.plugin.selfcontained.AbstractSelfContainedPlugin;
 import hec2.rts.plugin.RtsPlugin;
+import hec2.rts.plugin.RtsPluginManager;
 import hec2.rts.ui.RtsTabType;
+import sun.plugin2.main.server.Plugin;
 
 import java.util.List;
 
 public class FIRO_CFP_Plugin extends AbstractSelfContainedPlugin<FIRO_CFP_Alternative> implements RtsPlugin, CreatablePlugin {
+    public static final String PluginName = "FIRO_CFP";
+    private static final String _pluginVersion = "1.0.0";
+    private static final String _pluginSubDirectory = "cfp";
```

Comment:
In general, I find three letter acronyms to be confusing - what do you think about forecastprocessor as the folder directory?